fiftyone/core/utils.py

```diff
@@ -2064,6 +2129,280 @@ def get_alt_path(self, output_path, alt_dir=None):
         return os.path.join(root_dir, rel_path)
 
 
+def __rm_unique_filename_tmpdir():
+    with suppress(Exception):
+        shutil.rmtree(f"/tmp/fo-unq/{os.getpid()}")
+
+
+atexit.register(__rm_unique_filename_tmpdir)
+
+
+class MultiProcessUniqueFilenameMaker(object):
```

Comment:
I cannot review this also in this PR ... 😮‍💨

Reply:
https://github.com/voxel51/fiftyone/pull/5715